examples/AutomatedSummaryReport/main.bal

```diff
@@ -0,0 +1,55 @@
+import ballerina/io;
+import ballerinax/slack;
+
+string token = ?;
+
+type ChannelType record {
+    string id;    
+};
+
+type Channels record {|
+    boolean ok;
+    ChannelType[] channels;
+|};
+
+type TextType record{
+    string text;
+};
+
+type History record{
+    boolean ok;
+    TextType[] texts;
+};
+
+
+public function automatedStandUpReport() returns error?{
```

Comment:
please use vscode formatter to properly formate the file

Reply:
sure